php/templates/NOTES.txt

```diff
@@ -11,7 +11,7 @@
      NOTE: It may take a few minutes for the LoadBalancer IP to be available.
            You can watch the status of by running 'kubectl get svc -w {{ template "php.fullname" . }}'
   export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "php.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-  echo http://$SERVICE_IP:{{ .Values.service.port }}
+  echo http://$SERVICE_IP:80
```

Comment:
@cw-niinuma 
Will the endpoint port always be fixed at 80 in this PHP Chart?
`NOTES.txt` is always the way to display after `helm install`. 
Therefore, please display `NOTES.txt` including the installed contents.
https://helm.sh/docs/topics/chart_template_guide/notes_files/